editor/src/components/canvas/canvas-strategies/strategies/absolute-resize-bounding-box-strategy.spec.browser2.tsx

```diff
@@ -1569,32 +1614,127 @@ export var storyboard = (
+          import * as React from 'react'
+          import { Storyboard, Group } from 'utopia-api'
+
+          export var storyboard = (
+            <Storyboard data-uid='storyboard'>
+              <div data-uid='div' style={{ background: '#aaa', position: 'absolute', left: -50, top: 100, width: 300, height: 250 }}>
+                <Group data-uid='group' style={{ position: 'absolute', left: 50, top: 33, height: 89, right: 60, background: 'white' }}>
+                  <div data-uid='foo' style={{ position: 'absolute', width: 62, height: 56, background: 'red', top: 0, left: 0 }} />
+                  <div data-uid='bar' style={{ position: 'absolute', width: 62, height: 56, background: 'blue', top: 33, left: 128 }} />
+                </Group>
+              </div>
+            </Storyboard>
+          )
+        `),
+        )
+      })
+      it('when the group is pinned vertically', async () => {
+        const renderResult = await renderTestEditorWithCode(
+          formatTestProjectCode(`
+          import * as React from 'react'
+          import { Storyboard, Group } from 'utopia-api'
+
+          export var storyboard = (
+            <Storyboard data-uid='storyboard'>
+              <div data-uid='div' style={{ background: '#aaa', position: 'absolute', left: 100, top: 100, width: 200, height: 150 }}>
+                <Group data-uid='group' style={{ position: 'absolute', top: 50, bottom: 33, width: 89, left: 60, background: 'white' }}>
+                  <div data-uid='foo' style={{ position: 'absolute', width: 50, height: 56, background: 'red', top: 0, left: 0 }} />
+                  <div data-uid='bar' style={{ position: 'absolute', width: 50, height: 56, background: 'blue', top: 33, left: 100 }} />
+                </Group>
+              </div>
+            </Storyboard>
+          )
+        `),
+          'await-first-dom-report',
+        )
+
+        const target = EP.fromString('storyboard/div')
+
+        await renderResult.dispatch([selectComponents([target], false)], true)
+
+        await resizeElement(renderResult, { x: 0, y: -50 }, EdgePositionBottomRight, emptyModifiers)
+
+        expect(getPrintedUiJsCode(renderResult.getEditorState())).toEqual(
+          formatTestProjectCode(`
+          import * as React from 'react'
+          import { Storyboard, Group } from 'utopia-api'
+
+          export var storyboard = (
+            <Storyboard data-uid='storyboard'>
+              <div data-uid='div' style={{ background: '#aaa', position: 'absolute', left: 100, top: 100, width: 200, height: 100 }}>
+                <Group data-uid='group' style={{ position: 'absolute', top: 50, bottom: 33, width: 89, left: 60, background: 'white' }}>
+                  <div data-uid='foo' style={{ position: 'absolute', width: 30, height: 11, background: 'red', top: 0, left: 0 }} />
+                  <div data-uid='bar' style={{ position: 'absolute', width: 30, height: 11, background: 'blue', top: 6, left: 59 }} />
```

Comment:
I think the `width` of foo and bar should not change, only the height! can you change the code so that the starting group+children is already trued up?

Reply:
the idea was to have a non-trued-up group to make sure the true up works both ways, but it's fine by me to change it back :)

Reply:
oh okay, that's a fine idea then! but can you please add a comment about it? :)